allspice/utils/list_components.py

```diff
@@ -368,57 +451,109 @@ def _resolve_prjpcb_relative_path(schdoc_path: str, prjpcb_path: str) -> str:
 
 
 def _build_schdoc_hierarchy(
-    sheets_to_refs: dict[str, list[dict]],
+    document_jsons: dict[str, dict],
```

Comment:
We say `document` here, but I since we are expecting these all to be schematics we should change `schematic_jsons`

Reply:
Device Sheets are also technically schematics, which is why the distinction is between `Document` as in the project and `DeviceSheet` as in the project, but I can rename these to be schematic_documents to make that clearer.